cider/exec/nextgen/Nextgen.cpp

```diff
@@ -20,35 +20,43 @@
  */
 
 #include "exec/nextgen/Nextgen.h"
+#include <memory>
+#include "jitlib/base/JITFunction.h"
 
 namespace cider::exec::nextgen {
 
-std::unique_ptr<context::CodegenContext> compile(const RelAlgExecutionUnit& ra_exe_unit,
-                                                 const jitlib::CompilationOptions& co) {
-  auto context = std::make_unique<context::CodegenContext>();
-  jitlib::LLVMJITModule module("codegen", true, co);
+std::pair<std::unique_ptr<context::RuntimeContext>,
+          std::unique_ptr<context::CodegenContext>>
```

Comment:
It seems weird to return a `CodegenContext` after compile done. What will be included in it? Can the included info be moved to `RuntimeContext`?

Reply:
In my test, it will core dump if get RuntimeContext from CodegenContext after call compile function

Reply:
cause you move `LLVMModule` into `CodegenContext`, maybe should move into `LLVMJITFunction`